llvm/lib/DebugInfo/Symbolize/DIPrinter.cpp

```diff
@@ -313,18 +329,21 @@ void JSONPrinter::print(const Request &Request, const DIInliningInfo &Info) {
   json::Array Array;
   for (uint32_t I = 0, N = Info.getNumberOfFrames(); I < N; ++I) {
     const DILineInfo &LineInfo = Info.getFrame(I);
-    json::Object Object = toJSON(LineInfo);
+    json::Object Object = toJSON(LineInfo, I > 0);
     SourceCode SourceCode(LineInfo.FileName, LineInfo.Line,
```

Comment:
In llvm toJson was extracted, made the same change to reduce future merge conflicts